api/types.proto

```diff
@@ -747,3 +755,31 @@ message ManagerStatus {
 	// Reachability specifies whether this node is reachable.
 	RaftMemberStatus.Reachability reachability = 4;
 }
+
+// SecretReference is the linkage between a service and a secret that it uses.
+message SecretReference {
+	// Name is the name of the secret that this references.
```

Comment:
As discussed, this name will be informational, and `secret_id` below will be used to actually resolve the secret. Let's make this clear in the comments.